Index: src/main/scala-2.11/Main.scala
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/scala-2.11/Main.scala	(revision e095687828ca8baac92de4b7cebd6b4280eb5a70)
+++ src/main/scala-2.11/Main.scala	(revision )
@@ -1,3 +1,5 @@
 object Main {
 
+
+        case class Pair[T1, T2](t1: T1, t2: T2)
 }
